src/command_line_parser.cc

```diff
@@ -463,6 +462,14 @@ TritonParser::SetupOptions()
   http_options_.push_back(
       {OPTION_HTTP_THREAD_COUNT, "http-thread-count", Option::ArgInt,
        "Number of threads handling HTTP requests."});
+  http_options_.push_back(
+      {OPTION_HTTP_RESTRICTED_API, "http-restricted-api",
+       "<string>:<string>=<string>",
+       "Specify restricted HTTP api setting. The format of this "
+       "flag is --http-restricted-api=<apis>,<key>=<value>. Where "
+       "<api> is a comma-separated list of apis to be restricted. "
+       "<key> will be additional header key to be checked when a HTTP request "
+       "is received, and <value> is the value expected to be matched."});
```

Comment:
What do you think of including options here to avoid docs lookup?
```cpp
...
"Possible options are: " + Join(RESTRICTED_CATEGORY_NAMES, " ")
```

Same for grpc help text.

(I think I recently added a Join() helper to common.h)